templates/kubernetes/index.html

```diff
@@ -35,6 +35,27 @@ <h3>Fault Tolerant</h3>
   </div>
 </section>
 
+<section class="p-strip is-bordered">
+  <div class="row p-divider">
+    <div class="col-6 p-divider__block">
+      <h2>In partnership with Google&nbsp;GKE</h2>
+      <p>Google and Canonical together enable smooth hybrid operations between Google&rsquo;s Container Engine (GKE) service with Ubuntu worker nodes, and Canonical&rsquo;s Distribution of Kubernetes. Choose Canonical&rsquo;s Kubernetes to be sure your container workloads can migrate to GKE thanks to our kernel-to-k8s&nbsp;alignment.</p>
+      <p><a href="https://cloud.google.com/container-engine/docs/node-images#ubuntu_beta" class="p-link--external">Try GKE with Ubuntu now</a></p>
+      <div class="u-align--center u-hidden--small">
+        <img src="{{ ASSET_SERVER_URL }}ba70dd8a-gce.png?h=160" alt="Google Container Engine" />
+      </div>
+    </div>
+    <div class="col-6 p-divider__block">
+      <h2>In partnership with Rancher&nbsp;Labs</h2>
+      <p>Canonical, in partnership with Rancher Labs, delivers The Cloud Native Platform: a turn-key application delivery platform, built on Ubuntu, Kubernetes, and Rancher.  Maximize developer velocity integrated CI/CD, and ease the path from development into production with enterprise-caliber container management.</p>
+      <p><a href="#" class="p-link--external" onclick="dataLayer.push({'event' : 'GAEvent', 'eventCategory' : 'External Link', 'eventAction' : 'Watch webinar', 'eventLabel' : 'Watch webinar' : undefined });">Watch webinar [Need link]</a></p>
+      <div class="u-align--center u-hidden--small">
+        <img src="{{ ASSET_SERVER_URL }}8f878ba0-rancher-logo-stacked-color.svg" width="240" alt="" />
```

Comment:
If there is an alt on the logo above this should either have one or the Google one should be removed for consistency.